Makefile

```diff
@@ -63,7 +63,13 @@ test:
 		CACHE_ARGS=" --cache-from type=registry,ref=$(DOCKER_CACHE_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION}"; \
 	fi; \
 	test -n "$${PLUGIN_NAME}" -a -n "$${PLUGIN_VERSION}" && \
-	$(DOCKER) $(DOCKER_BUILD_ARGS) $(DOCKER_BUILD_EXTRA_ARGS)$${CACHE_ARGS} --build-arg PLUGIN_VERSION=$${PLUGIN_VERSION} -t $(DOCKER_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION} $(<D)
+	$(DOCKER) $(DOCKER_BUILD_ARGS) \
+		$(DOCKER_BUILD_EXTRA_ARGS)$${CACHE_ARGS} \
+		--build-arg PLUGIN_VERSION=$${PLUGIN_VERSION} \
+		--label build.buf.plugins.config.owner=$${PLUGIN_OWNER} \
+		--label build.buf.plugins.config.name=$${PLUGIN_NAME} \
```

Comment:
Added labels to match `buf alpha plugin push`. Going to remove the remote label from the published images since we use `--override-remote` and other things which can change what is in the buf.plugin.yaml.

Reply:
https://github.com/bufbuild/buf/pull/1362 updates labels when building an image to match.